app/server/lib/ICreate.ts

```diff
@@ -13,6 +13,9 @@ import {createSandbox, SpawnFn} from 'app/server/lib/NSandbox';
 import {SqliteVariant} from 'app/server/lib/SqliteCommon';
 import {ITelemetry} from 'app/server/lib/Telemetry';
 
+export const DEFAULT_SESSION_SECRET =
```

Comment:
This could do with a comment, there was a lot of discussion about when and whether secrets are needed and why they clearly don't matter now for new installations and didn't matter a lot with older installations. Could you do me a favor and capture a summary of this here? It is the most likely spot for someone scanning for hardcoded creds in an audit to be looking.

Reply:
How's that comment? Did I correctly understand the situation?

Reply:
Works for me, thank you.